src/components/switchButtons/SwitchButtons.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { useEffect, useState } from 'react';
+import cx from 'classnames';
+
+import { Container } from '../Container';
+import { createBemBlockBuilder } from '../../utils';
+
+import './SwitchButtons.scss';
+
+const BUTTON_WIDTH = 239;
+
+export const SwitchButtons = ({ buttons, onSwitch }) => {
+  const [switchState, setSwitchState] = useState(buttons);
+
+  const onSwitchButton = ({ btnName, active }) => {
+    if (active) {
+      return false
+    }
+
+    setSwitchState((previousSwitchState) => 
+      previousSwitchState.map((btn) => btnName === btn.text ? ({...btn, active: true}) : ({...btn, active: false}))
+    );
+
+    onSwitch(btnName);
+  };
+
+  const getWidth = () =>  BUTTON_WIDTH * switchState.length
```

Comment:
```suggestion
  const getWidth = () => BUTTON_WIDTH * switchState.length
```
Is prettier not handle it?